en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store uses an application that captures online orders. For each order, the application also needs to generate a notification for the sales manager as well as update the inventory recordsinventory records and notify the status to the client who places the order.
+
+## Before you begin
```

Comment:
Should we say as `Prerequisites` similar to https://wso2.com/choreo/docs/integrations/build-your-first-integration/